### ISSUE TITLE
Export annotation downloaded json file without annotation data  

### ISSUE THREAD

**Issue by ryan1799** [IDEA-Research/deepdataspace#163]
When the labeling is completed when the labeling is exported, select the coco format labeling file, after opening there is no labeling any data

**Comment by xifanii** [comment]
@ryan1799 Thank you for your feedback! 
To help us investigate further, could you please provide more details about the issue? Specifically:
- Are you using the quick annotation mode?
- What type of annotations are you working with?
- Could you share relevant screenshots of the interface and the exported files?
- 
Your input will greatly assist us in identifying the problem. Thank you!

**Comment by ryan1799** [comment]
Yes, I am using smart annotation mode with mask segmentation (interactive visual cues), after annotation all targets are successfully detected and segmented, after I save the annotation results and export the annotation, the coco.json file has no mask information.
![f103b2626a3d9e41d0fb7817ca058d9](https://github.com/user-attachments/assets/b6cac39f-dbfc-4922-8876-26cbf4e6be0f)


![fa65cc14bf6dfad995aebbc0bc261f1](https://github.com/user-attachments/assets/78798144-aca3-4529-8585-40882e657f8e)
![ef9c0e2db90f4e072b6434691fb0da9](https://github.com/user-attachments/assets/7fc9aa5c-66d9-49e0-b431-de86a59c91b1)
